docs/overrides/partials/integrations/analytics/custom.html

```diff
@@ -0,0 +1,10 @@
+<script async src="https://github.com/awslabs/clickstream-web/releases/download/{{ config.extra.analytics.version }}/clickstream-web.min.js"></script>
+<script>
+    window.onload = function() {
+        window.ClickstreamAnalytics.init({
+            appId: '{{ config.extra.analytics.appId }}',
+            endpoint: '{{ config.extra.analytics.endpoint }}',
```

Comment:
This is a multi-page application and we should add `pageType: window.PageType.multiPageApp` in order to track page views correctly.